### DISCUSSION TITLE
Regress out or not regress out, that is the question

### DISCUSSION THREAD

**Discussion by WesDe** [satijalab/seurat#4868] [General]
Hello,

I am working on an experiment composed of one cell type exposed to virus infection. I am comparing the non infected sample with the infected one.  I was wondering if regressing out by cell cycle and percent.mt made sense in this context. It is known that the virus used alters the cell cycle and can also induced higher cell death (higher percent.mt) when the virus is recognized.  

I have tried both integration methods (with and without regressing out). Without regression I can identify a well separate cluster associated with virus infection while using regression I have a unique big cluster that can be separated in multiple communities with one associated with viral infection.

Without regression:
![No_regrss](https://user-images.githubusercontent.com/18364708/127486509-deaa999e-0f8d-42c2-ac83-8864b2bdb21b.png)

With regression:
![Regress](https://user-images.githubusercontent.com/18364708/127486556-7d12654b-d168-4b94-a92c-dc6ea00a7085.png)

I would like the community opinion about the correct way to handle this type of data.

Best,
WesDe

**Comment by samuel-marsh** [comment]
So honestly I think there definitely isn't a "right" answer here and it's totally dataset and question dependent.  Cycling cells can often represent their own biology and not necessary appropriate to try and regress that factor out so that they cluster with others of same cell types.  But again it depends on what you want to convey in your visualization and what downstream analysis you will be performing.

For me personally I almost never regress cell cycle scores but again that's what I feel best fits my analysis and questions I'm asking.

Best,
Sam
